.eslintrc.js

```diff
@@ -17,7 +17,18 @@ module.exports = {
       { prefer: 'type-imports', fixStyle: 'inline-type-imports' },
     ],
     '@typescript-eslint/no-explicit-any': 'off',
-    '@typescript-eslint/no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-vars': [
+      'warn',
+      {
+        args: 'all',
+        argsIgnorePattern: '^_|\\$theme',
+        caughtErrors: 'all',
+        caughtErrorsIgnorePattern: '^_',
+        destructuredArrayIgnorePattern: '^_',
+        varsIgnorePattern: '^_',
+        ignoreRestSiblings: true,
```

Comment:
I took this configuration from the ts-eslint docs, but I think it would be useful if we need to omit a few props from an object.
https://eslint.org/docs/latest/rules/no-unused-vars#ignorerestsiblings